app/code/Magento/InventoryInStorePickupAdminUi/composer.json

```diff
@@ -0,0 +1,21 @@
+{
+    "name": "magento/module-inventory-in-store-pickup-admin-ui",
+    "description": "N/A",
+    "require": {
+        "php": "~7.1.3||~7.2.0",
+        "magento/framework": "*"
```

Comment:
please add the dependences.

`
"magento/module-inventory-in-store-pickup-api": "*",
"magento/module-backend": "*",
"magento/module-directory": "*",
"magento/module-ui": "*"
`

Reply:
@TadeuRodrigues Test fails due to:
3) Magento\Test\Integrity\DependencyTest::testRedundant
Redundant dependencies found!
Module Magento\InventoryInStorePickup: hard [Magento\InventoryInStorePickupApi] 
Module Magento\InventoryInStorePickupAdminUi: hard [Magento\Backend, Magento\Directory, Magento\InventoryInStorePickupApi, Magento\Ui]

Reply:
I'm checking.